admission/TAG

```diff
@@ -1 +1 @@
-0.9.3
+0.9.4
```

Comment:
Is this change intended to rebuild admission to reflect https://github.com/cybozu/neco-containers/pull/623/commits/6f831833c728ab9de5749f49d4f0b45ec9ad649a?

Reply:
Yes, it is needed to complete cybozu-go/neco#1638.